### DISCUSSION TITLE
problem with the optimizer

### DISCUSSION THREAD

**Discussion by Mosolim** [alexliniger/MPCC#65] [Q&A] [answered]
Hello All;
First thanks for the nice paper and code. 
I am running the code but unfortunately, I got an error using optimizer " too many input arguments"  while using optimization interface Yalmip or the other options 
BR
Mohamed

**Comment by fengyeying** [comment]
i got the same error..

**Comment by alexliniger** [comment, marked as answer]
Matlab introduced a function which is called "optimizer.m" which conflicts with the one of this repo. An easy fix is to rename optimizer.m to optimizer_mpcc.m and change everything accordingly. 

I will push the change ASAP
Best,
Alex

**Reply by alexliniger** [reply]
should be fixed now

**Comment by QQlearning007** [comment]
hi, what is the duty cycle d ?I don't understand it.

**Reply by alexliniger** [reply]
Can you please make this a new discussion entry? This helps other people find the answer. I will answer there.
Thanks,
Alex

**Comment by QQlearning007** [comment]
Ok, I'll try it right away.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alexliniger/MPCC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月22日(星期一) 晚上8:42
***@***.***&gt;;
***@***.******@***.******@***.***&gt;;
主题:&nbsp;Re: [alexliniger/MPCC] problem with the optimizer (Discussion #65)





 
Can you please make this a new discussion entry? This helps other people find the answer. I will answer there.
 Thanks,
 Alex
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
